task.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+require_once 'src/Services/DatabaseService.php';
+require_once 'src/Models/TasksModel.php';
+require_once 'src/Entities/TaskEntity.php';
+require_once 'src/Services/DateService.php';
+
+$db = DatabaseService::connect();
+$tasksModel = new TasksModel($db);
+
+if (isset($_GET['task']))
+{
+    $taskIdLink = $_GET['task'];
+} else
+{
+    $taskIdLink = 1;
+}
+
+$displayTask = $tasksModel->selectTaskById($taskIdLink);
+$displayTaskUser = $tasksModel->selectTaskUser($taskIdLink);
+$displayTaskUserName = $displayTaskUser->name;
+$displayTaskUserAvatar = $displayTaskUser->avatar;
+$dateNewFormat = DateService::reformatDateUK($displayTask->deadline);
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Project Manager - Project Name</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body class="h-full">
+<header class="p-3 bg-teal-50 flex justify-between">
+    <h1 class="sm:text-5xl text-4xl"><a href="index.php">Project Manager</a></h1>
+    <div class="pr-3 flex">
+        <a href=<?php echo "task.php?task=$taskIdLink"?> class="p-3 bg-slate-300 rounded-l-lg border-y border-l">🇬🇧</a>
+    </div>
+</header>
+<main class="p-3">
+    <div class="flex justify-between mb-3">
+        <h2 class="text-4xl font-bold mb-2"><?php echo $displayTask->name?>
+            <a href="project.php" class="text-base text-blue-600 hover:underline ms-3">Return to project</a>
+        </h2>
+        <div class="flex items-center gap-3">
+            <h3 class="text-3xl font-bold"><?php echo $displayTaskUserName?></h3>
+            <img class="w-[50px]" src=<?php echo $displayTaskUserAvatar?> alt="profile pic" />
```

Comment:
Same here - just use $displayTaskUser directly without the extra avatar variable